spectral/src/functions/allowPrometheusPlainText.ts

```diff
@@ -1,6 +1,42 @@
-// import { IFunctionResult } from "@stoplight/spectral-core";
+import { IFunctionResult } from "@stoplight/spectral-core";
+import { truthy, pattern } from "@stoplight/spectral-functions";
+import { isEqual } from "lodash";
 
-// // func that checks all fields return application/json content type except for a 200 GET response with the tag 'prometheus metrics
-// export default (targetVal: any, opts: any, context: any ): IFunctionResult[] => {
-//     console.log (targetVal, opts, context.path);
-//     }
\ No newline at end of file
+// func that checks all fields return application/json content type except for a 200 GET response with the tag 'prometheusMetrics'
```

Comment:
This is not finished yet, I'm just about done but I am having issue with different nested levels of js object literal